rust/worker/src/execution/orchestration/get_vectors.rs

```diff
@@ -58,6 +59,7 @@ impl ChromaError for GetVectorsError {
 }
 
 #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
is this orchestrator unused now?

Reply:
It's deprecated since we no longer implement the vector reader interface for query node.